### PR TITLE
Add IDL harness tests for motion sensors

### PR DESCRIPTION
**Pull request by Honry**
- Add IDL harness tests for Accelerometer, Gyroscope, Magnetometer
 - Add @Honry as test owner

### CODE REVIEW

File: accelerometer/idlharness.https.html [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+  required Error error;
+};
+
+</pre>
+
+<pre id="accelerometer-idl">
+[Constructor(optional AccelerometerOptions accelerometerOptions)]
+interface Accelerometer : Sensor {
+  readonly attribute AccelerometerReading? reading;
+  readonly attribute boolean includesGravity;
```
Comment (tobie) [inline accelerometer/idlharness.https.html]: `includesGravity` has still not been fixed in the spec?

Comment (Honry) [inline accelerometer/idlharness.https.html]: No, it's not yet, see the latest [ED](https://w3c.github.io/accelerometer/#accelerometer-interface).

Comment (Honry) [inline accelerometer/idlharness.https.html]: @tobie, can we merge it at first? As more new tests for motion sensors depend on this PR.
Once the API changes, I can help update tests immediately.

Comment (tobie) [inline accelerometer/idlharness.https.html]: I don't have an opinion on this, tbh.

**Comment by Honry** [comment]
Many thanks @tobie for the review, @dontcallmedom, could you help merge this PR?

**Comment by tobie** [comment]
Top directories need to match shortnames for the tooling to work. These should be in three directories.

**Comment by tobie** [comment]
@Honry: for the record, I did not review those tests. I just commented on one issue that jumped at me after a cursory glance.

At the very least, you want to split those tests up and put them in separate folders as they're separate specs.

**Comment by Honry** [comment]
@tobie, never mind. 😄

**Pull request merged**